modules/special-rules/partials/template-placement-bullet.adoc

```diff
@@ -0,0 +1,8 @@
+. Place a Barrage template so that it:
+* is completely within range and line of sight of the firer
```

Comment:
I suspect that you chose this phrasing to avoid the effective 3cm increase in the Range of the weapon from the radius of the template? I'd see this as a rules change, though not one that I am against in isolation.

However, this phrase also implies that no part of the template can be out of the line of fire -- which seems odd and overly restrictive. We could rephrase to fix this aspect; but then there is also the complication that 'line of fire' should incorporate weapon arcs, so there is a tension over whether or not the whole template must be inside the weapon arc too.

Is it perhaps better just to focus on the centre of the template and accept that template weapons get a 3cm range gain, and can be 3 cm outside of weapon arc? That's a little irritating I know, but I suspect that many players are used to it that way and accept it, it'd save us from writing more complex clauses for what will amount to rules changes in the eyes of many players I think.

---

Also, just so you know: As they stand, these partials work but they do add quite a lot of visual noise to the topics for something that soon becomes engrained in player's minds anyway. So, I'm experimenting with a different approach that uses the work you've done to separate out the sense but presents it in a more minimalistic way.

Reply:
I chose this phrasing because the Battles Book says:

> Barrage templates must be placed entirely within range and fire arc...

https://thehobby.zone/resources/e40k-compendium/Content/More/HouseRulesFrequentlyAskedQuestionsAndOtherNonsense/FAQ_Barrages.htm

It's a rules change in that I said 'line of sight' instead of 'fire arc', but I think it's going to get messy if we allow the template to be partially outside LoF/LoS when it's not allowed to be partially outside fire arc/weapon arc. Especially when there's a whole ability dedicated to shooting without line of fire.

Reply:
Ah, my bad, I relied on faulty memory when I dipped in and didn't check the original text. Pretty sure EA or some other GW stuff had the problems I was thinking of. 

Okay, that's actually good because it doesn't create the tensions I was thinking of. The only thing that still seems a bit odd to me is that in effect you may not target something through a gap if the shooting unit doesn't have line of fire to the entire area of the template. But if that's how it is, then that's fine I guess and all the more reason to have artillery.